includes/admin/marketplace-suggestions/class-wc-marketplace-updater.php

```diff
@@ -0,0 +1,71 @@
+<?php
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Marketplace Suggestions Updater
+ */
+class WC_Marketplace_Updater {
+
+	public static function load() {
```

Comment:
Missing docblock.

Reply:
Added docblocks for file & all functions here: c45a39676c509546d6e44469324de34d8f621f2d